lib/matplotlib/tests/test_axes.py

```diff
@@ -5091,11 +5091,26 @@ def test_broken_barh_empty():
     ax.broken_barh([], (.1, .5))
 
 
-def test_pandas_pcolormesh():
-    pd = pytest.importorskip('pandas')
-    from pandas.tseries import converter
-    converter.register()
+def _pandas_wrapper(func):
+    import functools
```

Comment:
Could this get a one line comment along the lines of "Decorator for tests that use pandas"?